react/AddToCartButton.tsx

```diff
@@ -144,44 +103,14 @@ const AddToCartButton: FC<Props & InjectedIntlProps> = ({
     showToast({ message, action })
   }
 
-  const [
-    addToCart,
-    { error: mutationError, loading: mutationLoading },
-  ] = useMutation<{ addToCart: OrderForm }, { items: OrderFormItemInput[] }>(
-    ADD_TO_CART
-  )
-
-  const beforeAddToCart = (event: React.MouseEvent) => {
+  const handleAddToCart: React.MouseEventHandler = event => {
     event.stopPropagation()
     event.preventDefault()
-  }
-
-  const callAddToCart = async () => {
-    const adjustedSkuItems = adjustItemsForMutationInput(skuItems)
 
-    const mutationResult = await addToCart({
-      variables: { items: adjustedSkuItems },
-    })
+    addItem(skuItems)
 
-    if (mutationError) {
-      console.error(mutationError)
-      toastMessage({ success: false, isNewItem: false })
-      return
-    }
-
-    if (
-      mutationResult.data &&
-      compareObjects(mutationResult.data.addToCart, orderForm)
-    ) {
-      toastMessage({ success: true, isNewItem: false })
-      return
-    }
```

Comment:
The toast message about an item which is already on the cart will not appear anymore?

Reply:
I don't know what to do in this case, because we don't have "feedback" about the result of the mutation, so there's currently no way we could know if the item was a duplicate or not.

Reply:
Unfortunately, this is kind of a deal-breaker :(
We need to show this duplicated item message. We didn't do it before, this was a feature we implemented in the legacy minicart a few months ago, removing it from the new one would be a regression.

Reply:
It should be doable to make our `addItem` function return some feedback so that we can maintain the current behavior of the `add-to-cart-button`, right?